setup.py

```diff
@@ -5,7 +5,7 @@
 except ImportError:
     from distutils.core import setup
 
-version = '0.5.12'
+version = '0.5.13'
```

Comment:
If this is a new feature (not a fix to existing feature, let's bump minor revision (0.6.0).

Reply:
Done.